### ISSUE TITLE
Incorrect `entity.handle()` call?

### ISSUE THREAD

**Issue by hjkatz** [mozman/ezdxf#12]
https://github.com/mozman/ezdxf/blob/master/ezdxf/query.py#L251

Should the call on this line be `entity.dxf.handle` instead?

Currently, I'm getting an error trying to accomplish the following:

```
import ezdxf
drawing = ezdxf.readfile("path/to/some/file.dxf")

q = modelspace().query('INSERT')
entity = q[0]

query = ezdxf.query.new()
query.extend(entity) # error, not a sequence
query.extend([entity]) # error, object has no attribute 'handle'
```

Basically, I'm trying to create a query from scratch and add only some entities to it that I already have reference too and this led me to this potential problem. Perhaps in addition to this fix there could be an addition of `query.add(Entity)`? Thoughts?

I'll create a pull request with the required work, I just need to know what you'd prefer :)

**Comment by mozman** [comment]
Thanks!

Fixed the error in EntityQuery.extend().

I think there is no need for an query.add() function, just collect the entities to add in a temporary list and add them at once.

Best regards,
Manfred

**Comment by hjkatz** [comment]
Awesome, thanks @mozman! Would you mind releasing this as a bug fix version as well? v0.7.10?

**Comment by mozman** [comment]
just released v0.8.0
